docs/reference/standards/guides/issuing-api-calls.md

```diff
@@ -1,5 +1,10 @@
 ---
```

Comment:
This doc should be put at the beginning of AvalancheGo section as this only applicable to AvalancheGo APIs, not others.

Reply:
Then a link to this should be put in those places where this doc is needed. However, I still see this approach convoluted.